playbooks/container-runtime/private/config.yml

```diff
@@ -9,6 +9,9 @@
   vars:
     l_default_container_runtime_hosts: "oo_nodes_to_config:oo_hosts_containerized_managed_true"
   roles:
+    - role: openshift_excluder
```

Comment:
Let's make this an import_role and make it the first task in tasks section.  The 'role: container_runtime" just imports default vars into the scope.

In the future, I'd like to see the excluders modified to import_role + tasks_from for each type of excluder.  As-is it's currently very confusing.  Also might make sense to just bundle this logic into the container runtime role itself.

Reply:
Done.

Agree, both excluders should be in separate playbooks, for now its configured by vars

Reply:
@vrutkovs you forgot to remove the role: openshift_exlcuder in the latest patch set.

Reply:
Oh, correct. Fixed